doc/man/md/vine_worker.md

```diff
@@ -90,6 +90,7 @@ OPTION_LONG(keep-workspace) Do not delete the contents of the workspace on worke
 - **--volatility=_&lt;chance&gt;_**<br />Set the percent chance per minute that the worker will shut down (simulates worker failures, for testing only).
 - **--connection-mode=_&lt;mode&gt;_**<br />When using -M, override manager preference to resolve its address. One of by_ip, by_hostname, or by_apparent_ip. Default is set by manager.
 - **--transfer-port=_&lt;port&gt;_**<br /> Listening port for worker-worker transfers.  (default: any))
+- **--transfer-address=_&lt;addr&gt;_**<br /> Explicit contact address for worker-worker transfers. (default: :<transfer_port>)
```

Comment:
Let's be very clear to distinguish between three things:
- `addr` to represent an IP address in string form.  (IPV4 or IPV6)
- `host` to represent a DNS domain name or address
- `hostport` to represent a DNS domain name/addr and port.

So I think this should be called `transfer-hostport`